### ISSUE TITLE
API is not supported anymore? return result is https://youtube.com/devicesupport 

### ISSUE THREAD

**Issue by 0xzkzhao** [kylejginavan/youtube_it#251]
When I do query by author, it returns this link: https://www.youtube.com/watch?v=UKY3scPIMd8
Does this mean the API is not supported anymore? 

How can I query for video now?

**Comment by adank92** [comment]
Same thing is happening to me, i'm guessing the code needs an update.

**Comment by adank92** [comment]
The library is using the deprecated Youtube API v2, that's why it's broken. There's a PR to add this to the README but the repo owner did not merge it yet: #210

I'm going to try this gem that supports v3: https://github.com/Fullscreen/yt

**Comment by 0xzkzhao** [comment]
@adank92  How are u doing with `yt`? I'm getting another error, have no idea about it.
https://github.com/Fullscreen/yt/issues/163

**Comment by adank92** [comment]
@cqcn1991 i've just tested in my machine the code that's giving your problems and it's working correctly.
#ruby --version
ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-linux
#rails --version
Rails 4.2.0
#rake --version
rake, version 10.4.2
#yt info
Yt version 0.22.0

**Comment by Mossie93** [comment]
I have similar problem. method `video_by` returns error 410, which causes app to render `status 500`

**Comment by tak1n** [comment]
http://youtube-eng.blogspot.co.at/2015/04/bye-bye-youtube-data-api-v2.html

**Comment by h0jeZvgoxFepBQ2C** [comment]
Yeah, this gem doesnt work anymore.. Please merge the pull request.

**Comment by kterusaki** [comment]
Same here. I can't access youtube resources via youtube_it gem, but can via other gems.

**Comment by rubyFeedback** [comment]
Is there an alternative to this abandoned gem here?
